browser/actors/FathomChild.jsm

```diff
@@ -13,6 +13,11 @@ var EXPORTED_SYMBOLS = ["FathomChild"];
 //   "resource://gre/modules/Fathom.jsm"
 // );
 
+const COLORS = {
+  shopping: "red",
+  article: "blue",
+}
```

Comment:
;

Reply:
Huh; eslint didn't show anything until I used the `./mach eslint` command for the first time. I pushed directly to master to [fix the eslint errors](https://github.com/erikrose/gecko-dev/commit/14289e9a78b0be564ee9df62adca2b115064f333).